s3cloudfront/README.md

```diff
@@ -0,0 +1,33 @@
+# FightPandemics Content Delivery Infrastructure
+
+This repo contains the Terraform code for setting up the FightPandemics Content Delivery infrastructure.
+
+Note that the Terraform state for this module is separate from the main infrastructure Terraform state.
+
+## Prerequisites
+
+1. Install Terraform v0.12.29 by downloading the package for your operating system [here](https://www.terraform.io/downloads.html). Alternatively, you can run Terraform in [Docker](https://hub.docker.com/r/hashicorp/terraform). NOTE: Please install Terraform v0.12.29 as v0.13 has some breaking changes.
+
+1. In the `#devops` Slack channel, ask for the following:
+- Access to this repository (provide your Github username)
+- Access to our sandbox development AWS account (provide your email address)
+
+## Contributing and Testing in Dev environment
+
+1. Clone this repo, and create a new branch.
+
+1. Change directory into the `s3cloudfront` folder in this repo.
+
+1. Copy `fp_context.cdn.env.example` to `dev.env`. In the `#devops` Slack channel, ask for the proper values to add to this file for the development environment. Please note that the AWS credentials are session-based, and therefore they are short-lived. 
```

Comment:
This should probably be `development` to be consistent with the other Terraform code.